lib/MusicBrainz/Server/Entity/Role/Age.pm

```diff
@@ -38,13 +38,16 @@ sub has_age
             )
         ) == -1;
 
-    # If there is no end date, then the end date is now() (so there is an age).
-    my @end_comp = $self->end_date->defined_run or return 1;
```

Comment:
This incorrectly thought of entities that have ended at an unknown date as non-ended.